docs/source/guides/building_bentos.rst

```diff
@@ -0,0 +1,241 @@
+.. _bento-format-page:
+
+Building Bentos
+===============
+
+Bento is a standardized file archive format in BentoML that describes
+how to load and run a ``bentoml.Service`` defined by the user. It includes
+code that instantiates the ``bentoml.Service`` instance, as well
+as related configurations, data/model files, and dependencies.
+
+A Bento can be built with the ``bentoml build`` command with the ``bentofile.yaml``
+configuration file. Here's an example of that process from the `quickstart guide (https://github.com/bentoml/gallery/tree/main/quickstart#build-bento-for-deployment)_`.
+
+.. code:: yaml
+
+   service: "service:svc"
+   description: "file: ./README.md"
+   labels:
+      owner: bentoml-team
+      stage: demo
+   include:
+      - "*.py"
+   python:
+      packages:
+         - scikit-learn
+         - pandas
+
+The service field is the python module that holds the bentoml.Service
+instance.
+
+Configuring files to include
```

Comment:
lets make a table including a brief explanation of common fields that could have under a `bentofile.yaml`. You can put the include section in that table as well.

Reply:
like an anatomy of a bentofile

Reply:
that's great idea, maybe only put the top level fields here in a table?

Reply:
agreed